docs/host-page-integration.md

```diff
@@ -0,0 +1,29 @@
+# Interacting with the sidebar
+
+## Show the sidebar:
+
+It is possible to show the Hypothesis sidebar from a third party page
```

Comment:
Same for the count section below. Instead of offering answers to questions with "it is possible to..." reiterate the question they may have with the answer itself.